<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Android::Internal::AddNewAVDDialog</class>
 <widget class="QDialog" name="Android::Internal::AddNewAVDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>243</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new AVD</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="abiLabel">
       <property name="text">
        <string>Architecture (ABI):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="sdcardSizeLabel">
       <property name="text">
        <string>SD card size:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="targetApiLabel">
       <property name="text">
        <string>Target API:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="deviceDefinitionLabel">
       <property name="text">
        <string>Device definition:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="abiComboBox"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QSpinBox" name="sdcardSizeSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> MiB</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="targetApiComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="deviceDefinitionComboBox"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="Utils::InfoLabel" name="warningText"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="deviceDefinitionTypeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="overwriteCheckBox">
       <property name="text">
        <string>Overwrite existing AVD name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::InfoLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/infolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Android::Internal::AddNewAVDDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Android::Internal::AddNewAVDDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
